code/frameworks/preact-vite/src/typings.d.ts

```diff
@@ -0,0 +1,43 @@
+// TODO: Replace, as soon as @types/react-dom 17.0.14 is used
+// Source: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/fb0f14b7a35cde26ffaa82e7536c062e593e9ae6/types/react-dom/client.d.ts
+declare module 'react-dom/client' {
```

Comment:
Not sure whether this applies to preact? Above my pay grade 🤷

Reply:
I simplified back to the `preact-webpack5` typings.